### ISSUE TITLE
return dimensions of custom loss functions

### ISSUE THREAD

**Issue by b-biswas** [b-biswas/MADNESS#5]
in the file [FlowVAEnet.py](https://github.com/b-biswas/FlowDeblender/blob/main/scripts/FlowVAEnet.py) the custom loss function takes input of dimension: [batch_size, 59, 59, 6] where 59,59,6 is an image with 6 channels. 
Currently, I use tf.redue_mean so the return dimension is [batch_size] and the code seems to be working well. 

In the original debvader repo, Bastien was performing the same operation but he was not using tf.reduce_sum, so his return dimension was [batch_size, 59, 59, 6]. This did not throw an error but the model was not able to train (The errors in prediction were huge) 

@junpenglao can you confirm if it is ok the way I am doing it? or should I take the mean to return a scalar?

**Comment by junpenglao** [comment]
The loss fun should return a scalar (taking the mean should be fine), since the batch represent data batch but not model batch

**Comment by b-biswas** [comment]
The training is working now!
